improver/psychrometric_calculations/cloud_condensation_level.py

```diff
@@ -19,6 +19,52 @@
     dry_adiabatic_temperature,
     saturated_humidity,
 )
+from improver.utilities.common_input_handle import as_cubelist
+
+
+class MetaPluginCloudCondensationLevel(PostProcessingPlugin):
+    """
+    Meta-plugin which handles the calling of HumidityMixingRatio followed by CloudCondensationLevel.
+    Derives the temperature and pressure of the convective cloud condensation
+    level from near-surface values of temperature, pressure and humidity mixing
+    ratio.
+    """
+
+    def __init__(self, model_id_attr: str = None):
+        """
+        Set up class
+
+        Args:
+            model_id_attr:
+                Name of model ID attribute to be copied from source cubes to output cube
+        """
+        from improver.psychrometric_calculations.psychrometric_calculations import (
+            HumidityMixingRatio,
+        )
```

Comment:
The motivation was in enforcing that a science configuration would have to reference a plugin from the module where it is defined, rather than from some other module that uses it.  I'll move the import to the top at this time.  Can peruse this potential issue in future.

Reply:
OK, I wouldn't have considered this possibility. Is this your motivation for the change in `test_MetaPluginCloudCondensationLevel.py` to use 
```
@patch(
    "improver.psychrometric_calculations.cloud_condensation_level.HumidityMixingRatio"
)
```
rather than 
```
@patch(
    "improver.psychrometric_calculations.psychrometric_calculations.HumidityMixingRatio"
)
```
even though the `HumidityMixingRatio` plugin is defined within psychrometric_calculations.py?

Why would someone do this?

Reply:
> Is this your motivation for the change in test_MetaPluginCloudCondensationLevel.py

When it comes to patching, that is a different matter (it is the `MetaPluginCloudCondensationLevel` class under test which requires its objects from the cloud_condensation_level to be patched).

My concern was more around a science configuration specifically.  If someone references a plugin from a module that uses it, to execute, then that creates an implicit dependency on this other module.  That is, if someone changes that module, it could break their configuration even though they aren't intending to make changes to the plugin itself from the module where it is defined.
This opens up a wider discussion around discoverability etc. which is likely beyond the scope of this PR.  CLIs are all available in a flat space namespace.  That is, all under `improver.cli`
Similarly, I think it might be useful to make plugins available under a flat namespace (thereby protecting science configurations from changes in project layout/structure).  For example, perhaps `improver.plugin.MetaPluginCloudCondensationLevel`.
As I say, I think this is beyond the scope of this PR and warrants its own PR so as not to detract from the changes of this work.

Reply:
OK, I can understand the concern from a technical perspective where this kind of implicit dependency could be created. In practice, I'm not aware that we ever import things like that to create an implicit dependency, but I can see that it would be possible. I think that your comments about plugin discoverability are interesting (#2005) and probably a good idea.